### PR TITLE
feat: adding rds disk encryption option

### PR DESCRIPTION
**Pull request by deimosfr**

### CODE REVIEW

**Comment by deimosfr** [comment]
Waiting for core team validation before merging

File: src/models.rs
Comment (erebe) [inline src/models.rs:612]: Why prefix the field with database_ ? We are already in the database object

Comment (deimosfr) [reply src/models.rs:612]: To have consistency with others. But I agree we can remove it as it's already in databse

**Review by benjaminch — APPROVED** [review]

**Comment by erebe** [comment]
Please ensure we set in the core DB all flag encryption to false when this PR is merged

**Pull request merged**
